web/src/main/resources/static/js/tree.js

```diff
@@ -335,8 +321,8 @@ var tree = (function () {
                 }
 
                 var nci_links = [];
-                if (typeof d.nci !== 'undefined' && d.nci != '') {
-                    $.each(d.nci.split(","), function( index , value ) {
+                if (typeof d.nci !== 'undefined' && d.nci.length > 0) {
+                    $.each(d.nci, function( index , value ) {
```

Comment:
another good simplification .. not relying on the presentation style of lists from the tumort_types.txt API endpoint. This must have required some searching during testing to find cases with multiple NCI codes - nice.